tests/components/plex/test_config_flow.py

```diff
@@ -434,10 +442,14 @@ async def test_option_flow_new_users_available(
     OPTIONS_OWNER_ONLY[MP_DOMAIN][CONF_MONITORED_USERS] = {"Owner": {"enabled": True}}
     entry.options = OPTIONS_OWNER_ONLY
 
-    mock_plex_server = await setup_plex_server(config_entry=entry)
+    mock_plex_server = await setup_plex_server(config_entry=entry, disable_gdm=False)
+
+    # Multiple debouncers with cooldowns involved, easier to trigger manually
+    async_dispatcher_send(hass, PLEX_GDM_CLIENT_SCAN_SIGNAL)
```

Comment:
As soon as we're relying on integration details in the tests they become less robust. It would be better to call the plex websocket callback to trigger this signal. We seem to do that already below. Why isn't that enough?

Reply:
We can move time forward as needed by firing a time changed event.
https://github.com/home-assistant/core/blob/5dbb5f12eba9f88a507bf6ca480356e2fc717c30/tests/common.py#L287-L288

Reply:
Seems this was a race between dispatching a call to begin the GDM scan and checking the result of that scan before it was scheduled/completed.